apps/synapse-portal-framework/src/components/NavLink.tsx

```diff
@@ -1,35 +1,44 @@
+import { Link as MuiLink } from '@mui/material'
 import React from 'react'
-import { Link, LinkProps } from 'react-router-dom'
+import { NavLink as ReactRouterNavLink } from 'react-router-dom'
 
 /*
   
-  Surprisingly the Link component only supports internal web app links, so its necessary
+  Surprisingly the ReactRouterNavLink component only supports internal web app links, so its necessary
   to check if the link being passed in is external and use an explicit anchor tag.
   
   Read more here: https://github.com/ReactTraining/react-router/issues/1147
 
 */
 
-type Props = LinkProps & {
-  text: string | JSX.Element
+type Props = {
+  to: string
+  onClick?: () => void
+  className?: string
+  id?: string
+  style?: React.CSSProperties
+  children: React.ReactNode
+  target?: string
 }
 export default function NavLink(props: Props) {
   const link = props.to as string
-  const target = props.target ?? '_self' // default to open in the same window, if this is an external link
-  const isExternal = () => {
-    if (link.includes('http')) {
-      return true
-    } else if (link.includes('mailto')) {
-      return true
-    }
-    return false
+  let defaultTarget = '_self' // default to open in the same window
+  let isExternal = false
+  if (link.startsWith('http')) {
+    defaultTarget = '_blank' // if this is an external link, default to open in a new window
```

Comment:
Change the default behavior (I don't think there is any case today where we use this for an external link and don't add `target="_blank"`)